src/test/java/api/support/builders/BatchChangeDueDateRequestBuilder.java

```diff
@@ -0,0 +1,61 @@
+package api.support.builders;
+
+import org.folio.circulation.support.JsonPropertyWriter;
+
+import io.vertx.core.json.JsonArray;
+import io.vertx.core.json.JsonObject;
+import java.util.ArrayList;
+import java.util.List;
+import org.joda.time.DateTime;
+
+public class BatchChangeDueDateRequestBuilder extends JsonBuilder implements
+  Builder {
+
+  private final List<String> loanIds;
+  private final DateTime dueDate;
+  private boolean emptyBody;
+
+  public BatchChangeDueDateRequestBuilder() {
+    this(new ArrayList<>(), null);
+    emptyBody = false;
+  }
+
+  public BatchChangeDueDateRequestBuilder(List<String> loanIds,
+    DateTime dueDate) {
+    this.loanIds = loanIds;
+    this.dueDate = dueDate;
+  }
+
+  @Override
+  public JsonObject create() {
+    final JsonObject payload = new JsonObject();
+
+    if (!emptyBody) {
+      JsonPropertyWriter.write(payload, "loanIds", new JsonArray(loanIds));
+      JsonPropertyWriter.write(payload, "dueDate", dueDate);
```

Comment:
Personally, I would use a static import to improve the readability of these lines